src/sass/shared/variables.scss

```diff
@@ -208,10 +223,13 @@ $size-codepills-project-height: 30px;
 $size-codepills-project-width: $size-codepills-raw-width * $size-codepills-project-height / $size-codepills-raw-height;
 $size-codepill-project-vertical-correction: $size-codepills-project-height * $size-codepill-raw-vertical-correction / $size-codepills-raw-height;
 
+$tall-line-height: 1.2em;
```

Comment:
`line-height` seems to be defined under its own section
https://github.com/src-d/landing/blob/master/src/sass/shared/variables.scss#L100